Tests/DatadogTests/Datadog/RUM/RUMMonitor/RUMCommandTests.swift

```diff
@@ -43,4 +46,28 @@ class RUMCommandTests: XCTestCase {
             """
         )
     }
+
+    func testWhenRUMAddCurrentViewErrorCommand_isPassedErrorSourceTypeAttribute() {
+        let command1 = RUMAddCurrentViewErrorCommand(time: .mockAny(), error: SwiftError(), source: .source, attributes: [RUMAttribute.internalErrorSourceType: "react-native"])
+
+        XCTAssertEqual(command1.errorSourceType, .reactNative)
+        XCTAssertTrue(command1.attributes.isEmpty)
+
+        let command2 = RUMAddCurrentViewErrorCommand(time: .mockAny(), message: .mockAny(), type: .mockAny(), stack: .mockAny(), source: .source, attributes: [RUMAttribute.internalErrorSourceType: "react-native"])
+
+        XCTAssertEqual(command2.errorSourceType, .reactNative)
+        XCTAssertTrue(command2.attributes.isEmpty)
+    }
+
+    func testWhenRUMStopResourceWithErrorCommand_isPassedErrorSourceTypeAttribute() {
+        let command1 = RUMStopResourceWithErrorCommand(resourceKey: .mockAny(), time: .mockAny(), error: SwiftError(), source: .source, httpStatusCode: .mockAny(), attributes: [RUMAttribute.internalErrorSourceType: "react-native"])
+
+        XCTAssertEqual(command1.errorSourceType, .reactNative)
+        XCTAssertTrue(command1.attributes.isEmpty)
+
+        let command2 = RUMStopResourceWithErrorCommand(resourceKey: .mockAny(), time: .mockAny(), message: .mockAny(), type: .mockAny(), source: .source, httpStatusCode: .mockAny(), attributes: [RUMAttribute.internalErrorSourceType: "react-native"])
+
+        XCTAssertEqual(command2.errorSourceType, .reactNative)
+        XCTAssertTrue(command2.attributes.isEmpty)
+    }
```

Comment:
We're missing the case of using `.ios` default value when `RUMAttribute.internalErrorSourceType` is not specified.